src/main/java/core/basesyntax/operation/SupplyOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperation implements OperationHandler {

```

Comment:
Don't begin class implementation with an empty line. According to the checklist, remove all redundant empty lines.